apps/src/music/views/MusicView.jsx

```diff
@@ -614,6 +654,25 @@ const MusicView = connect(
           state.progress.currentLessonId
         ).findIndex(level => level.isCurrentLevel)
       : state.music.currentProgressState.step,
+
+    // When we are in a lesson with multiple levels, they are here.
+    levels: state.progress.lessons
+      ? levelsForLessonId(state.progress, state.progress.currentLessonId)
+      : undefined,
+
+    // When we are editing a single level, this will be its ID.
```

Comment:
Is this comment still true? This would be set even if we're in a lesson with multiple levels right?